tools/clang/test/CodeGenDXIL/hlsl/intrinsics/linalg_builtins/check-shader-stages.hlsl

```diff
@@ -0,0 +1,131 @@
+// RUN: %dxc -T lib_6_9 %s | FileCheck %s
+ 
+ByteAddressBuffer matrix_buffer;
+ByteAddressBuffer bias_buffer;
+RWByteAddressBuffer rw_matrix_buffer;
+
+void UseCoopVec() {
+	vector<float, 4> output_vector;
```

Comment:
OMG these tests have _tabs_!
_gasp_
/me: clutching pearls

Seriously though, can you do a replace all `/t` with `  `?

We don't really apply the coding standards to tests, but if we're not actually testing the ability of the compiler to parse odd things it is nice to have at least some consistency of style.

Reply:
Yep, replaced all "`\t`" with "`    `". Resolved.